libs/sr-formula/src/data/common/index.ts

```diff
@@ -0,0 +1,37 @@
+import { max, min, prod } from '@genshin-optimizer/pando'
+import type { TagMapNodeEntries } from '../util'
+import { percent, reader, self, selfBuff } from '../util'
+import dmg from './dmg'
+import prep from './prep'
+
+const data: TagMapNodeEntries = [
+  ...dmg,
+  ...prep,
+
+  reader.withTag({ src: 'iso', et: 'self' }).reread(reader.src('custom')),
+  reader.withTag({ src: 'agg', et: 'self' }).reread(reader.src('custom')),
+
+  // convert src:char, lightCone, relic to src:agg for accumulation
+  reader.src('agg').add(reader.sum.src('char')),
```

Comment:
According to the table in `data/util/tag.ts`, do we also include `src:iso <= sum { src:char }` somewhere?

Reply:
Think this is it? https://github.com/frzyc/genshin-optimizer/blob/3badff12590f8e253dc17a72f7fe0548a6ab3444/libs/sr-formula/src/util.ts#L28